src/Core/SettingsChangesHistory.cpp

```diff
@@ -98,6 +98,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"validate_enum_literals_in_operators", false, false, "A new setting"},
             {"allow_experimental_kusto_dialect", true, false, "A new setting"},
             {"allow_experimental_prql_dialect", true, false, "A new setting"},
+            {"h3togeo_standard_order", false, true, "New setting, fixes h3ToGeo() function to return (lat, lon)"},
```

Comment:
Just saying `New setting`, like for the other entries, is fine. Further explanation (like you did) only duplicates what is already mentioned in Core/Settings.cpp.